code/modules/mob/living/living_defines.dm

```diff
@@ -6,19 +6,19 @@
 	var/resize = RESIZE_DEFAULT_SIZE //Badminnery resize
 
 	//Health and life related vars
-	var/maxHealth = 100 //Maximum health that should be possible.
-	var/health = 100 	//A mob's health
-	var/health_threshold_dead = -100
-	var/health_threshold_crit = 0
+	var/maxHealth = 100 ///Maximum health that should be possible.
+	var/health = 100 	/// Mob's current health
+	var/health_threshold_dead = -100 /// Health at which a mob dies
```

Comment:
```suggestion
	///Maximum health that should be possible.
	var/maxHealth = 100
	/// Mob's current health
	var/health = 100 
	/// Health at which a mob dies
	var/health_threshold_dead = -100
```
This is the format in where the comments will show.

Reply:
It works in both cases,, and for vars like this i'd prefer it inline, unless the line is already massive.

![image](https://user-images.githubusercontent.com/1134201/79485478-39bf7400-800d-11ea-9dcf-4b1eaa291a25.png)

Reply:
it doesnt work properly if you have
```
/type
    var/foo /// doc comment for var

/**
  * doc comment for proc
 */
/type/proc/bar()
```

Reply:
That sucks, okay ill migrate to the line before

<details>
<summary>For those interested how it turns out</summary>
Code:

```

/**
* Page comment for spooky.dm
*/
/mob/spooky
	var/testing = TRUE  /// This sucks

/**
* This is the proc doc comment
*/
/mob/spooky/proc/test_proc(var/arg_one)
	return testing /// Returns testing

```

Docs

![image](https://user-images.githubusercontent.com/1134201/79497008-e81fe500-801e-11ea-86ba-16913dac285d.png)

</details>